coral-spark/src/test/java/com/linkedin/coral/spark/CoralSparkTest.java

```diff
@@ -464,4 +464,16 @@ public void testLateralViewMapWithoutColumns() {
     assertEquals(CoralSpark.create(relNode).getSparkSql(), targetSql);
   }
 
+  @Test
+  public void testConcat() {
+    RelNode relNode = TestUtils.toRelNode("SELECT 'a' || 'b'");
+
+    String targetSql = "SELECT concat('a', 'b')\nFROM (VALUES  (0)) t (ZERO)";
```

Comment:
`FROM (VALUES  (0)) t (ZERO)` is redundant. In some cases, errors can occur.
Will you directly transfer to SQL? @ljfgem @wmoustafa I thought you only used view, so I just closed it myself. I can commit my code if necessary

Reply:
Yes, our usage scenario is to convert Hive SQL to Trino SQL. I'm a little curious about what case will make mistakes. Can you tell me the details, thanks @JiajunBernoulli . And I don't think the definition of coral is just used to convert view.

Reply:
@JiajunBernoulli, it could translate view or pure Hive SQL. Actually, for views, we also retrieve the its Hive SQL and translate.
And could you please elaborate why do you think `FROM (VALUES (0)) t (ZERO)` is redundant and error-prone? Feel free to commit your code. Thanks!

Reply:
@ljfgem @huangxiaopingRD The problem I encountered occurred in the `union`, I checked it, there's no problem now. Sorry to waste your time.